spec/controllers/ems_middleware_controller_spec.rb

```diff
@@ -17,7 +17,15 @@
       session[:settings] = {:views => {}, :quadicons => {}}
       EvmSpecHelper.create_guid_miq_server_zone
       login_as FactoryGirl.create(:user)
-      @middleware = FactoryGirl.create(:ems_hawkular)
+      auth = AuthToken.new(
```

Comment:
do we have a test that tests the appearance of this in the summary page itself? afaik the below is only adding the auth to the mock (which is great by itself as well :+1: )